chia/wallet/util/query_filter.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from enum import IntEnum
+from typing import List
+
+from chia.types.blockchain_format.sized_bytes import bytes32
+from chia.util.ints import uint8, uint64
+from chia.util.streamable import Streamable, streamable
+from chia.wallet.util.transaction_type import TransactionType
+
+
+class FilterMode(IntEnum):
+    include = 1
+    exclude = 2
+
+
+class CoinRecordOrder(IntEnum):
```

Comment:
Why did you also move this one into here? It's not really related to the filtering, is it?